settings.ini

```diff
@@ -1,32 +1,23 @@
 [DEFAULT]
-# All sections below are required unless otherwise specified
 lib_name = fastai2_audio
 user = rbracco
 description = Audio Module for fastai version 2
 keywords = audio, deep learning, machine learning, fastai
 author = Your Name
 author_email = email@example.com
-# copyright = Your Name or Company Name
 branch = master
 version = 0.0.1
-min_python = 3.6
+min_python = 3.7
 audience = Developers
 language = English
-# Set to True if you want to create a more fancy sidebar.json than the default
 custom_sidebar = False
-# Add licenses and see current list in `setup.py`
 license = apache2
-# From 1-7: Planning Pre-Alpha Alpha Beta Production Mature Inactive
 status = 2
-# Change to, e.g. "nbs", to put your notebooks in nbs dir instead of repo root
 nbs_path = nbs
 doc_path = docs
-# Optional. Same format as setuptools console_scripts
-# console_scripts = 
-# Optional. Same format as setuptools requirements
-requirements = fastai2 fastcore torchaudio>=0.3 librosa colorednoise soundfile
-# Anything shown as '%(...)s' is substituted with that setting automatically
-git_url = https://github.com/%(user)s/%(lib_name)s/tree/%(branch)s/
-lib_path = %(lib_name)s
-title = %(lib_name)s
-
+requirements = torchaudio>=0.3.1 fastai2 fastcore librosa colorednoise soundfile pillow==6.1 matplotlib scikit-learn scipy spacy
+git_url = https://github.com/rbracco/fastai2_audio/tree/master/
+lib_path = fastai2_audio
+title = fastai2_audio
+doc_host = https://github.com/rbracco/fastai2_audio
```

Comment:
I believe doc_host needs to be changed to https://rbracco.github.io/fastai2_audio and that should do it